libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -238,11 +238,11 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
     }
 
     @ReactMethod
-    fun receivePayment(amountSats: Double, description: String, promise: Promise) {
+    fun receivePayment(reqData: ReceivePaymentRequest, promise: Promise) {
         executor.execute {
             try {
-                val payment = getBreezServices().receivePayment(amountSats.toULong(), description)
-                promise.resolve(readableMapOf(payment))
+                val response = getBreezServices().receivePayment(reqData)
+                promise.resolve(readableMapOf(response))
             } catch (e: SdkException) {
                 e.printStackTrace()
                 promise.reject(e.javaClass.simpleName, e.message, e)
```

Comment:
With the ReadableMap we need to convert it to a ReceivePaymentRequest type.
```
            val receivePaymentRequest = asReceivePaymentRequest(reqData)

            if (receivePaymentRequest == null) {
                promise.reject(GENERIC_CODE, "Invalid reqData")
            } else {
                try {
                    val response = getBreezServices().receivePayment(receivePaymentRequest)
                    promise.resolve(readableMapOf(response))
                } catch (e: SdkException) {
                    e.printStackTrace()
                    promise.reject(e.javaClass.simpleName, e.message, e)
                }
            }
```

Reply:
Done in 3b9ef403befd5079c2e761d6dacd0f0beda0e125